frontend/app/component/form/polling_station/PollingStationForm.tsx

```diff
@@ -77,13 +95,24 @@ export function PollingStationForm({ electionId, pollingStation, onSaved, onCanc
                 fieldWidth="narrow"
                 defaultValue={pollingStation?.number}
                 pattern="[0-9]*"
```

Comment:
To prevent the default tooltip warning related to the pattern @jorisleker noticed in his comment, you have to set the `title` attribute on fields with a pattern with a custom text. See https://www.w3schools.com/TAGS/att_input_pattern.asp

Reply:
I think that will still cause the browser tooltip to appear, just with a custom text.

If we want to disable the browser validation completely we can add the [`novalidate` attribute to the `<form>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#novalidate).

Reply:
The novalidate is indeed added to the form, but that means that you can type anything in the field with the pattern for numbers only. If I put `1a` in the field, it will save successfully and remove the `a`, instead of say it's not a number. I think that's not the behaviour we want right?

Reply:
How do we do this in the data entry forms? There, you can enter only numbers, and we don't show the default browser tooltip

Reply:
Ignore previous comment, I see this issue has been fixed. I don't see the tooltip anymore, and the correct error message is shown below the input:
![Scherm­afbeelding 2024-12-20 om 09 23 23](https://github.com/user-attachments/assets/3c8f6dc3-3669-4632-9967-719ea6ce6ab8)

Reply:
data entry form use NumberInput which has custom key press handlers.
We could do that here as well, just the same but without formatting. 

Personally I would prefer to use `<input type="number" />` but this is up for debate.

Reply:
The `<input type="number" />` is not an option design wise because you cannot turn off the up and down arrows. Meaning you can accidentally change the number when you press arrows on your keyboard, which is not desired.

Reply:
I removed the `pattern` and `novalidate` because together they do nothing.